plugins/catalog-react/src/components/EntityDisplayName/EntityDisplayName.tsx

```diff
@@ -77,7 +86,9 @@ export const EntityDisplayName = (
   );
 
   // The innermost "body" content
-  let content = <>{primaryTitle}</>;
+  let content = (
```

Comment:
A [tooltip](https://v4.mui.com/components/tooltips/#tooltip) would be nice to show the entire name when you hover over the link.

Reply:
ToolTip has been added now 
<img width="1710" alt="Screenshot 2024-10-18 at 7 41 51 PM" src="https://github.com/user-attachments/assets/05a06f83-472a-485c-a029-e1cd7877c575">